From: David Paleino <d.paleino@gmail.com>
Date: Mon, 9 Nov 2009 09:22:12 +0000
Subject: 0013 Fix man pages

Fix man pages, manpage-has-errors-from-man and hyphen-used-as-minus-sign

Forwarded: no
---
 ispell.1X | 2 +-
 ispell.5X | 6 +++---
 2 files changed, 4 insertions(+), 4 deletions(-)

diff --git a/ispell.1X b/ispell.1X
index cf49735..ff479ae 100644
--- a/ispell.1X
+++ b/ispell.1X
@@ -1326,7 +1326,7 @@ If the
 the output is made visually cleaner (but harder to post-process)
 by changing it to:
 .IP
--strip+add<tab>count<tab>bytes
+\-strip+add<tab>count<tab>bytes
 .PP
 where
 .IR strip ,
diff --git a/ispell.5X b/ispell.5X
index 6da54ad..3cfcda1 100644
--- a/ispell.5X
+++ b/ispell.5X
@@ -141,8 +141,8 @@ This feature can be used to convert an entire dictionary if necessary:)
 	echo qqqqq > dummy.dict
 	buildhash dummy.dict \fIaffix-file\fP dummy.hash
 	awk '{print "*"}END{print "#"}' \fIold-dict-file\fP \e
-	| ispell -a -T \fIold-dict-string-type\fP \e
-	  -d ./dummy.hash -p ./\fInew-dict-file\fP \e
+	| ispell \-a \-T \fIold-dict-string-type\fP \e
+	  \-d ./dummy.hash \-p ./\fInew-dict-file\fP \e
 	  > /dev/null
 	rm dummy.*
 .fi
@@ -1088,7 +1088,7 @@ is useful, as in the following example:
 .PP
 .RS
 .nf
-$ munchlist -c oldaffixes -l newaffixes olddict > newdict
+$ munchlist \-c oldaffixes \-l newaffixes olddict > newdict
 .fi
 .RE
 .PP
